### DISCUSSION TITLE
CefSharp jumps back from specific page to origin

### DISCUSSION THREAD

**Discussion by Gather9182** [cefsharp/CefSharp#4122] [General]
Hello!
I am new with working with CefSharp.
I am trying to use it in order to scrape information from a webstore. Currently everything works, but navigating to a specific page on a website seems to not work. The browser automatically remove the suffix "?page=2" from the url.
More precise I am talking about this URL: https://shop.billa.at/warengruppe/getraenke/spirituosen/B2-35
Here I want to load the page https://shop.billa.at/warengruppe/getraenke/spirituosen/B2-35?page=2

With everyother browser I can directly navigate to this page.
CefSharps seems to not work though.

/Edit: I am using CefSharp Offscreen

Thanks in advance!

**Comment by amaitland** [comment]
What version are you using? WinForms or WPF or OffScreen? 

Check the log file and DevTools for errors.

- https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file
- https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#javascript-debugging

**Reply by Gather9182** [reply]
Offscreen CefSharp.
I dont believe we are talking an error here. It just navigates back.

**Reply by amaitland** [reply]
Version? 

Please check the log file and DevTools. There's more than just errors logged.

**Reply by amaitland** [reply]
Please also test with the WPF version of the `MinimalExample` available at https://github.com/cefsharp/CefSharp.MinimalExample

**Reply by Gather9182** [reply]
> Please also test with the WPF version of the `MinimalExample` available at https://github.com/cefsharp/CefSharp.MinimalExample
Regarding this. The WPF version, as well as the WinForms one, actually loads the page correctly.

The Logfile shows:
`530/082457.949:WARNING:frame_impl.cc(431)] SendProcessMessage sent to detached frame 21474836484 [5,4] will be ignored
[0530/082457.949:WARNING:frame_impl.cc(431)] OnDidFinishLoad sent to detached frame 21474836484 [5,4] will be ignored
[0530/082457.949:WARNING:frame_impl.cc(431)] SendProcessMessage sent to detached frame 21474836484 [5,4] will be ignored`

Regarding the DevTools, I have no idea what to look for in them.

**Reply by amaitland** [reply]
> Regarding this. The WPF version, as well as the WinForms one, actually loads the page correctly.

Great. Are you have a custom `RequestHandler`? Often this sort of problem is a simple mistake in a custom `RequestHandler`.

> Regarding the DevTools, I have no idea what to look for in them.

Check the console for errors. Look at the `Network Tab` to see how the request.

**Reply by amaitland** [reply]
You still haven't answered as to which version you are using....

**Reply by Gather9182** [reply]
Thank you so much for your replies.
It feels bad to say but I made a typo in my URL. That explains everything.
